src/genie/libs/parser/dell/tests/ShowMacAddressTable/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,79 @@
+expected_output = {
```

Comment:
no big deal. another way would be having 'vlan` as top level.
```
'vlan': {
     '10': {
          'macs': {
```
so that we can easily pickup macs only under `vlan 10`.